content/en/docs/setup/_index.md

```diff
@@ -112,4 +112,5 @@ The following production environment solutions table lists the providers and the
 | [VMware](https://cloud.vmware.com/) | [VMware Cloud PKS](https://cloud.vmware.com/vmware-cloud-pks)              |[VMware Enterprise PKS](https://cloud.vmware.com/vmware-enterprise-pks)        |   [VMware Enterprise PKS](https://cloud.vmware.com/vmware-enterprise-pks)          | [VMware Essential PKS](https://cloud.vmware.com/vmware-essential-pks)      |          |[VMware Essential PKS](https://cloud.vmware.com/vmware-essential-pks)
 | [Z.A.R.V.I.S.](https://zarvis.ai/) | &#x2714; | | | | | |
 
+If you want to do it yourself, follow the documentation by starting with [installing kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
```

Comment:
Shouldn't this be part of the table above?

Reply:
Installing it yourself is a special case wrt to the table so I feel it makes more sense to explicitly mention it.

Reply:
How about both? Add the kubeadm option to the table _and_ have a section about installing your own production-grade cluster.

Reply:
Even better!

Reply:
@beamdee if you add that entry into the table as well, this will be good to merge.